src/Structs/Playlist.js

```diff
@@ -0,0 +1,9 @@
+import Track from './Track';
+
+export default class Playlist {
+  constructor(playlistObject, id) {
```

Comment:
It's strange that the playlist object doesn't have its own id as an attribute =_=